File: zero/Maruko.Zero/Internal/Domain/SysUser.cs
using FreeSql.DataAnnotations;
using Maruko.Core.FreeSql.Internal;

namespace Maruko.Zero
{
    [Table(Name = "sys_user")]
    public class SysUser : FreeSqlEntity
    {
        [Column(Name = "roleId")]
        public long RoleId { get; set; }

        [Column(Name = "userName", StringLength = 32)]
        public string UserName { get; set; }

        [Column(Name = "password", StringLength = 500)]
        public string Password { get; set; }

        [Column(Name = "icon", StringLength = 500)]
        public string Icon { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/domain/SysUser.java
package com.act.modules.zero.internal.domain;

import com.act.core.domain.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("sys_user")
@SuppressWarnings("all")
public class SysUser extends BaseEntity<Long> {

    //角色Id
    @TableField("roleId")
    private Long roleId;

    //用户名
    @TableField("userName")
    private String userName;

    //密码
    @TableField("password")
    private String password;

    //头像
    @TableField("icon")
    private String icon;
}
